tempesta_fw/work_queue.h

```diff
@@ -30,7 +30,7 @@ typedef struct {
 } __WqItem;
 
 #define WQ_ITEM_SZ		sizeof(__WqItem)
-#define TFW_WQ_CHECKSZ(t)	BUILD_BUG_ON(sizeof(t) != WQ_ITEM_SZ)
+#define TFW_WQ_CHECKSZ(t)	BUILD_BUG_ON(sizeof(t) > WQ_ITEM_SZ)
```

Comment:
It seems the change is wrong: different item sizes can lead to memory corruptions.